src/pages/app/sidebar/ProjectFilters.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import Link from "@/router/Link";
+  import { allDocumentsUrl, userUrl, orgUrl } from "@/search/search";
+  import { orgsAndUsers } from "@/manager/orgsAndUsers";
+  import { _ } from "svelte-i18n";
+</script>
+
+<style>
+  .linksection {
+    margin: 20px 0 40px 0;
+  }
+
+  .linksection :global(a) {
+    display: block;
+    padding: 5px 24px;
+    font-size: 14px;
+    color: rgba(0, 0, 0, 0.8);
+  }
+
+  .linksection :global(a).active {
+    /* $primary-faded */
+    background: rgba(66, 148, 240, 0.13);
+  }
+
+  /* $mobileBreak */
+  @media screen and (max-width: 720px) {
+    .linksection :global(a) {
+      /* (24px + $sidebarAdd) */
+      padding: 5px 24px 5px 48px;
+    }
+  }
+</style>
+
+<div class="linksection">
+  <Link plusReplace={true} toUrl={allDocumentsUrl()}>
+    <div class="link">{$_("projects.allDocuments")}</div>
+  </Link>
+  {#if $orgsAndUsers.me != null}
```

Comment:
Not that it should stop this from going out, but this should probably be `!==` for consistency and because we know that `getMe()` returns that if anything goes wrong.

Reply:
Most of the code uses `!=` now, but I agree with you that we should be using `!==` and `===` everywhere. We'll get there.